conda_smithy/linter/lints.py

```diff
@@ -1168,3 +1168,18 @@ def lint_recipe_is_parsable(
                     hints.append(
                         f"The recipe is not parsable by parser `{parser_name}`. {msg}"
                     )
+
+
+IS_AB3_BOOL_RE = re.compile(r"is_abi3\s(==|!=)\s('|\")(true|false)('|\")")
```

Comment:
somebody once told me that one can breakup a regexp into multiple lines in order to reduce the information density of the regexp

That way you can have "one idea per line"

Reply:
The `\s`es here should have a `?` (or better: `*`), because without that, you'll miss the cases where someone left out one of the spaces, .e.g. `is_abi3=="true"`